src/plugins/plot/MctPlot.vue

```diff
@@ -276,9 +282,21 @@ export default {
 
     },
     beforeDestroy() {
+        document.removeEventListener('keydown', this.handleKeyDown);
```

Comment:
Should these be debounced?

Reply:
I think it would be a good idea to add denounce. 
Though similar code was used on Conductor.vue line 141,142. Should I change both files?

Reply:
I tried to implement the debounce but it creates some unexpected bugs since I have 2 event simultaneously working (keyDown keyUp). I think it might not be necessary in this case.

Reply:
Hi @shefalijoshi, let me know if I have to make further changes. Thanks!